src/Editor/Resources_BASE_7056.cpp

```diff
@@ -0,0 +1,210 @@
+#include "Resources.hpp"
```

Comment:
This whole file should get deleted, we don't want merge and BASE files in our master.